### PR TITLE
hardening/77_tls_support

### PR DESCRIPTION
**Pull request by frbattid**
* Implements issue #77 
* No unit tests nor e2e tests are required.
* Assignee @gtorodelvalle

### CODE REVIEW

File: cosmos-gui/src/app.js [hunk 1]
```diff
@@ -49,6 +51,10 @@ var scEndpoint = config.clusters.storage.endpoint;
 var ccPrivKey = config.clusters.computing.private_key;
 var ccUser = config.clusters.computing.user;
 var ccEndpoint = config.clusters.computing.endpoint;
+var httpsOptions = {
+    key: fs.readFileSync(config.private_ley_filekey),
```
Comment (gtorodelvalle) [inline cosmos-gui/src/app.js]: I think there's a typo here: `private_ley_filekey` -> `private_key_file` :) Apart from this, I may have asked this before :p but have you confirmed that this works if `config.private_ley_filekey` and `certificate_file` are blank (`''`) as in the default configuration? :)

Comment (frbattid) [inline cosmos-gui/src/app.js]: Yes, there is a typo, I'll fix it.

Regarding the parameters check, I have a tech [debt issue](https://github.com/telefonicaid/fiware-cosmos/issues/30) about it... that I think I will implement in the second release :)

Comment (frbattid) [inline cosmos-gui/src/app.js]: Fixed in https://github.com/telefonicaid/fiware-cosmos/commit/6c6563e7fd8d41a80d9bbbd4ac8091ccfcfca831

**Comment by gtorodelvalle** [comment]
LGTM and merging! :) Thanks @frbattid

**Pull request merged**